### DISCUSSION TITLE
Lose delimiter during uploadind CSV

### DISCUSSION THREAD

**Discussion by stepanof** [nocodb/nocodb#4139] [Q&A]
Hello team!

I can't understand if I found a bug or something is missconfigurated.
I have csv file like this:
```
string;string2;string3
ok;10,5;okey
```
Also I created table in NocoDB:

![image](https://user-images.githubusercontent.com/68236577/196986790-180d7dc5-d75b-4f31-82ae-d971c13ef53a.png)
Here is 3 columns with SingleLineText type.

Then I try to Upload csv file.

![image](https://user-images.githubusercontent.com/68236577/196987066-c5b0692a-4859-47bc-9291-c4c9e455abfd.png)

After uploading I see lost comma in the second column.

![image](https://user-images.githubusercontent.com/68236577/196987220-4303e599-9e0a-4ee7-9035-4de8bcd45ba1.png)

**Even If I change the second colimns type to Decimal the same error occur.**

Is it a bug?

PS
sample csv file is attached
[number.csv](https://github.com/nocodb/nocodb/files/9831451/number.csv)

**Comment by wingkwong** [comment]
Can you share your project info?

**Reply by stepanof** [reply]
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.98.1**

**Reply by wingkwong** [reply]
Able to reproduce. Let's track [here](https://github.com/nocodb/nocodb/issues/4145).
